AppCenter/AppCenterTests/MSChannelUnitDefaultTests.m

```diff
@@ -109,7 +109,7 @@ - (void)testPendingLogsStoresStartTimeWhenPaused {
   [self initChannelEndJobExpectation];
   id dateMock = OCMClassMock([NSDate class]);
   NSObject *object = [NSObject new];
-  NSDate *date = [[NSDate alloc] initWithTimeIntervalSince1970:3000];
+  __block NSDate *date = [[NSDate alloc] initWithTimeIntervalSince1970:3000];
```

Comment:
The `date` is used in a block but I don't know why it didn't complain at compile time.
The change makes sense to me but I still don't know the change fixes the main problem.

Reply:
+1

@dtrabo please run CI a few more times to make sure that it helps